ui-service/src/app/component/user-form/user-form.component.html

```diff
@@ -16,9 +16,16 @@
             <input formControlName="object" matInput required placeholder="Object"/>
             <mat-error *ngIf="objectFc.hasError('required')">Object is required.</mat-error>
           </mat-form-field>
+          <mat-form-field>
+            <mat-label>Path length</mat-label>
+            <mat-select formControlName="pathlength">
+              <mat-option value="2">2</mat-option>
+              <mat-option value="3">3</mat-option>
+            </mat-select>
+          </mat-form-field>
         </div>
         <div>
-          <mat-checkbox formControlName="verbalize" matTooltip="Enabling verbalization will cause the request to take more time">Verbalized</mat-checkbox>
+          <mat-checkbox formControlName="virtualType" matTooltip="Enabling virtualType">Use Virtual Type</mat-checkbox>
```

Comment:
That seems like the verbalization has been removed from the UI. Is that necessary? It would be better to keep the feature :thinking:

Reply:
based on this [line](https://github.com/dice-group/COPAAL/blob/9f91844f23b235d4bc9c0d0c27e23b61aff74e2f/service/src/main/java/org/dice/FactCheck/Corraborative/Config/ConfigController.java#L46)     
I considered we don't need this anymore but based on yesterday meeting I should consider some changes to use verbalization

Reply:
now we have verbalization too :)